src/main/java/org/jabref/gui/entryeditor/SciteTab.java

```diff
@@ -0,0 +1,122 @@
+import org.controlsfx.control.HyperlinkLabel;
+
+public class SciteTab extends EntryEditorTab {
+
+    public static final String NAME = "Scite";
+
+    private final GridPane sciteResultsPane;
+    private final ProgressIndicator progressIndicator;
+    private final SciteTabViewModel viewModel;
+    private final PreferencesService preferencesService;
+
+    public SciteTab(PreferencesService preferencesService, TaskExecutor taskExecutor) {
+        this.preferencesService = preferencesService;
+        this.viewModel = new SciteTabViewModel(preferencesService, taskExecutor);
+        this.sciteResultsPane = new GridPane();
+        this.progressIndicator = new ProgressIndicator();
+
+        setText(Localization.lang("Scite"));
+        setTooltip(new Tooltip(Localization.lang("Search scite.ai for Smart Citations")));
+        setSciteResultsPane();
+    }
+
+    private void setSciteResultsPane() {
+        progressIndicator.setMaxSize(100, 100);
+        sciteResultsPane.add(progressIndicator, 0, 0);
+
+        ColumnConstraints column = new ColumnConstraints();
+        column.setPercentWidth(100);
+        column.setHalignment(HPos.CENTER);
+
+        sciteResultsPane.getColumnConstraints().setAll(column);
+        sciteResultsPane.setId("scitePane");
+        setContent(sciteResultsPane);
+
+        EasyBind.subscribe(viewModel.statusProperty(), status -> {
+            sciteResultsPane.getChildren().clear();
+            switch (status) {
+                case IN_PROGRESS:
+                    sciteResultsPane.add(progressIndicator, 0, 0);
+                    break;
+                case FOUND:
+                    if (viewModel.getCurrentResult().isPresent()) {
+                        sciteResultsPane.add(getTalliesPane(viewModel.getCurrentResult().get()), 0, 0);
+                    }
+                    break;
+                case ERROR:
+                    sciteResultsPane.add(getErrorPane(), 0, 0);
+                    break;
+            }
+        });
+    }
+
+    @Override
+    public boolean shouldShow(BibEntry entry) {
+        return viewModel.shouldShow();
+    }
+
+    @Override
+    protected void bindToEntry(BibEntry entry) {
+        viewModel.bindToEntry(entry);
+    }
+
+    private VBox getErrorPane() {
+        Label titleLabel = new Label(Localization.lang("Error"));
+        titleLabel.setStyle("-fx-font-size: 1.5em;-fx-font-weight: bold;-fx-text-fill: -fx-accent;");
+        Text errorMessageText = new Text(viewModel.searchErrorProperty().get());
+        VBox errorMessageBox = new VBox(30, titleLabel, errorMessageText);
+        errorMessageBox.setStyle("-fx-padding: 30 0 0 30;");
+        return errorMessageBox;
+    }
+
+    private VBox getTalliesPane(SciteTabViewModel.SciteTallyDTO tallyDTO) {
+        Label titleLabel = new Label(Localization.lang("Tallies for") + " " + tallyDTO.getDoi());
+        titleLabel.setStyle("-fx-font-size: 1.5em;-fx-font-weight: bold;");
```

Comment:
Please move all padding and style stuff to the EntryEditor.css so it can be tweaked from the outside